### PR TITLE
Add logger to yeoman generator

### PR DESCRIPTION
**Pull request by doug-wade**
Demonstrate how to use the logger.  Addresses #209

### CODE REVIEW

File: packages/generator-react-server/generators/app/templates/hello-world.js [hunk 1]
```diff
@@ -1,10 +1,14 @@
 import React from 'react';
+import rs from 'react-server';
```
Comment (gigabo) [inline packages/generator-react-server/generators/app/templates/hello-world.js]: I've been leaning toward this convention:

```javascript
import {logging} from 'react-server';
```

Thoughts?

File: packages/generator-react-server/package.json [hunk 2]
```diff
@@ -44,6 +44,7 @@
     ],
     "envs": [
       "node"
-    ]
+    ],
+    "globals": ["__LOGGER__", "__CACHE", "__CHANNEL__"]
```
Comment (gigabo) [inline packages/generator-react-server/package.json]: Why the other globals?  They're not used, right?  Just `__LOGGER__`?

File: packages/generator-react-server/generators/app/templates/package.json [hunk 3]
```diff
@@ -28,6 +28,7 @@
   "xo": {
     "esnext": true,
     "extends": "xo-react",
-    "ignores": ["__clientTemp/**/*"]
+    "ignores": ["__clientTemp/**/*"],
+    "globals": ["__LOGGER__", "__CACHE", "__CHANNEL__"]
```
Comment (gigabo) [inline packages/generator-react-server/generators/app/templates/package.json]: If this were alphabetized it would be pure addition. 👹

**Comment by gigabo** [comment]
Looks good.  Thanks @doug-wade. 👍

**Comment by doug-wade** [comment]
Here's what it looks like
screen shot 2016-05-25 at 5 42 25 pm

**Pull request merged**